test/unittests/CandidateTest.php

```diff
@@ -515,6 +515,7 @@ public function testLastRecordChangeBy()
      */
     public function testGetListOfVisitLabels()
     {
+        $this->markTestSkipped("Test is incomplete");
```

Comment:
I had to uncomment the assertEquals line because otherwise "$expected" is unused, but then the test fails because it seems to have never worked properly. (The select in getListOfVisitLabels was never mocked, there's just a line comment saying to mock it in the test..)